packages/amplication-server/src/core/subscription/subscription.service.ts

```diff
@@ -84,8 +84,7 @@ export class SubscriptionService {
     workspaceId: string
   ): Promise<void> {
     if (plan !== EnumSubscriptionPlan.Free && userId) {
-      await this.analyticsService.track({
-        userId: userId,
+      await this.analyticsService.trackWithContext({
         properties: {
           workspaceId: workspaceId,
```

Comment:
Not sure I understand the "rules" when should we pass in the workspaceId and when not (since trackWithContext adds it anyway)

Reply:
good catch. The workspace must be sent if there is no context. In this case, the wrong method was used (committing change)